idl/net.idol

```diff
@@ -110,10 +110,13 @@ Interface(
         ),
         "get_mac_address": (
             doc: "Reports the MAC address of port 0",
-            reply: Result(
-                ok: "MacAddress",
-                err: ServerDeath,
-            ),
+            reply: Simple("MacAddress"),
+            idempotent: true,
+        ),
+        "get_leftover_mac_addresses": (
+            doc: "Reports any remaining MAC addresses after the net task has claimed its own; e.g., leftover macs from local VPD for use by the host on a gimlet",
```

Comment:
Same as below, I'm not sure this is Gimlet-specific!

Reply:
Agreed; I'm happy to either remove the example from the doc comment or extend it ("for use by the host on a gimlet or the tofino on a sidecar" or somesuch). Do you have a preference between those?

Reply:
I'd vote to remove the example entirely

Reply:
Removed in ed0e851